group_vars/all/osa.yml

```diff
@@ -58,6 +58,15 @@ nova_console_type: novnc
 # RabbitMQ overrides
 rabbitmq_ulimit: 65535
 
+rabbitmq_policies:
+  - name: "HA"
+    pattern: '^(?!amq\.).*'
+    tags:
+      ha-mode: exactly
+      ha-params: 2
```

Comment:
I see no need to have more than 2 copies even when using 5 rabbitmq nodes. The quorum has nothing to do with the number of message copies, there is no correlation.